cas-server-core/src/main/java/org/jasig/cas/util/DefaultCipherExecutor.java

```diff
@@ -153,9 +145,13 @@ private String encryptValue(@NotNull final String value) {
      */
     private String decryptValue(@NotNull final String value) {
         try {
-            final Key key = new AesKey(this.encryptionSeedArray);
+            final Map<String, Object> keys = new HashMap<>(2);
```

Comment:
This code (4 lines) is duplicated from the previous method. What's the magic between `keys.put("kty", "oct");` ? Cannot we have some constants to help understanding?
Can this initialization happen outside the encryption/decryption methods?